### PR TITLE
Gently manually reformat code. Add missing imports.

### PR DESCRIPTION
**Pull request by petroolg**

### CODE REVIEW

File: ucats/baselines.py [hunk 1]
```diff
@@ -307,7 +306,7 @@ def first_pc_baseline(frames,
 def multi_scale_simple_baseline(y,
                                 plow=50,
                                 th=3,
-                                smooth_levels=[10, 20, 40, 80, 160],
```
Comment (petroolg) [inline ucats/baselines.py:310]: Mutable function parameter

File: ucats/detection1d.py [hunk 2]
```diff
@@ -182,7 +183,7 @@ def simple_pipeline_nojitter_(y, tau_label=1.5):
     Detect and reconstruct Ca-transients in 1D signal
     """
     ns = rolling_sd_pd(y)
-    low = y < 2.5 * np.median(y)
+    low = y < (2.5 * np.median(y))
```
Comment (petroolg) [inline ucats/detection1d.py:186]: Less ambiguous :)

**Review by petroolg — COMMENTED** [review]
I've only added stuff, didn't remove anything (apart from couple of brackets here an there).

File: ucats/detection1d.py (hunk 2 above)
Comment (abrazhe) [reply ucats/detection1d.py:186]: yes, thank you

File: ucats/baselines.py (hunk 1 above)
Comment (abrazhe) [reply ucats/baselines.py:310]: You are right!

**Comment by abrazhe** [comment]
Fanstastic! Merging.

**Pull request merged**